src/modules/accounting/accounting.py

```diff
@@ -11,6 +13,8 @@
 class Accounting(BaseModule, ConsensusModule):
     def __init__(self, w3: Web3):
         self.report_contract = w3.lido_contracts.accounting_oracle
+        ConsensusModule.report_contract = self.report_contract
+        BunkerService.report_contract = self.report_contract
```

Comment:
Set global attributes?
If you want to set this attributes, it's better to just pass them into __init__

Reply:
I'll remove it